d2l-scroll-wrapper.html

```diff
@@ -11,135 +14,335 @@
 
 			:host([h-scrollbar]),
-			:host-context([dir="rtl"])[scroll-rtl-reverse] {
+			:host-context([dir="rtl"])[scroll-rtl-reverse][h-scrollbar],
+			:host-context([dir="rtl"])[scroll-rtl-negative][h-scrollbar] {
 				@apply(--d2l-scroll-wrapper-h-scroll);
 			}
 
 			:host-context([dir="rtl"])[scroll-rtl-reverse][scrollbar-right],
+			:host-context([dir="rtl"])[scroll-rtl-negative][scrollbar-right],
 			:host([scrollbar-left]) {
 				@apply(--d2l-scroll-wrapper-left);
 			}
 
 			:host-context([dir="rtl"])[scroll-rtl-reverse][scrollbar-left],
+			:host-context([dir="rtl"])[scroll-rtl-negative][scrollbar-left],
 			:host([scrollbar-right]) {
 				@apply(--d2l-scroll-wrapper-right);
 			}
+
+			.action {
+				@apply(--d2l-scroll-wrapper-action);
+				@apply(--d2l-scroll-wrapper-action-hidden);
+			}
+
+			.end {
+				@apply(--d2l-scroll-wrapper-action-end);
+			}
+
+			.start {
+				@apply(--d2l-scroll-wrapper-action-start);
+			}
+
+			/*
+			 * So, this is where things get interesting
+			 * On Chrome, the scrolling logic is the same when direction is RTL
+			 * On Firefox, the scrolling becomes negative when direction is RTL
+			 * On IE, the scrolling is reversed when direction is RTL
+			 * When direction is RTL, the SVG icons are reversed
+			 *
+			 * I address these cases by having 3 groups of buttons. The group is displayed given the correct case
+			 * .default is displayed when the scrollbar is visible for all browsers in LTR. It is the sane case
+			 * .reverse is the same as default, except the buttons are swapped. Only displayed when scrolling is reversed (IE)
+			 * .negative is the inverse of default. Displayed when the scrolling direction is the same as in the LTR case (Firefox, Chrome)
+			 */
+			:host([h-scrollbar]) .default,
+			:host-context([dir="rtl"])[h-scrollbar][scroll-rtl-negative] .negative,
+			:host-context([dir="rtl"])[h-scrollbar][scroll-rtl-default] .negative,
+			:host-context([dir="rtl"])[h-scrollbar][scroll-rtl-reverse] .reverse {
+				@apply(--d2l-scroll-wrapper-action-visible);
+			}
+
+			/* Hide the start/end buttons depending on the state */
+			:host([scrollbar-right]) .end,
+			:host([scrollbar-left]) .start,
+			/* This is needed to hide the LTR case when direction is RTL */
+			:host-context([dir="rtl"])[h-scrollbar] .default,
+			/* needed for the increased selector specificity */
+			:host-context([dir="rtl"])[scroll-rtl-default][scrollbar-right] .end,
+			:host-context([dir="rtl"])[scroll-rtl-reverse][scrollbar-right] .start,
+			:host-context([dir="rtl"])[scroll-rtl-negative][scrollbar-right] .start,
+			:host-context([dir="rtl"])[scroll-rtl-default][scrollbar-left] .start,
+			:host-context([dir="rtl"])[scroll-rtl-reverse][scrollbar-left] .end,
+			:host-context([dir="rtl"])[scroll-rtl-negative][scrollbar-left] .end {
+				@apply(--d2l-scroll-wrapper-action-hidden);
+			}
 		</style>
+		<button class="default start action" is="d2l-image-action" icon="[[startIcon]]" on-tap="handleTapBack"></button>
```

Comment:
I expect it here, which is why I have all the different buttons for the different RTL cases. The CSS actually hides the LTR buttons and uses swapped buttons when `dir="rtl"`.

I suppose I could check if `dir="rtl"` in the `handleTapLeft` and `handleTapRight` buttons